configs/sst_program-unwanted.yaml

```diff
@@ -50,6 +50,7 @@ data:
     - koji-utils
     - koji-vm
     - koji-web
+    - pandoc
```

Comment:
This doesn't look correct.  I understand pandoc in Fedora is built so that it requires GHC, but in RHEL we build it differently so it doesn't get GHC pulled in.

@yselkowitz is there a way to stub out pandoc with a placeholder package to reflect how we build it in RHEL instead of marking it unwanted entirely?

Reply:
Nevermind, I am mis-remembering what we did in RHEL 9.  This is correct.